builder/comp-builder.nix

```diff
@@ -146,7 +146,6 @@ stdenv.mkDerivation ({
         (builtins.trace "WARNING: license \"${package.license}\" not found" license-map.LicenseRef-OtherLicense);
   };
 
-  CABAL_CONFIG = configFiles + /cabal.config;
```

Comment:
Isn't this going to affect all the component builds too? I'd have thought we only wanted to get rid of this for the shell builder?

Reply:
This variable is used by `cabal-install` only. It was there for when running something like `nix-shell -A hsPkgs.mypackage.components.library`.